bedrock/firefox/templates/firefox/new/trailhead/thanks-a.html

```diff
@@ -0,0 +1,83 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros-protocol.html" import hero, picto_card with context %}
+
+{% extends "firefox/new/trailhead/base.html" %}
```

Comment:
The download isn't working because there's no JS bundle included in the template here. If you extend the regular `thanks.html` template then it should work again, without having to redefine things.
```suggestion
{% extends "firefox/new/trailhead/thanks.html" %}
```